### DISCUSSION TITLE
[Feature request] One level checkbox nesting

### DISCUSSION THREAD

**Discussion by Korben11** [GitJournal/GitJournal#939] [Feature Requests]
In Google Keep Notes, it's possible to drag and drop checkbox items into groups:

 - [ ] Items group
   - [x] group item 1
   - [ ] group item 2
 - [ ] Next item
 - [ ] Next items group
   - [ ] item

The advantage of this is the ability to categorize items.   
Also, this way, it allows you to check and uncheck the whole group simultaneously.

**Comment by vHanda** [comment]
Moving this to Discussion as it isn't a bug.
